packages/react-components/src/components/styles/MessageThread.styles.ts

```diff
@@ -118,6 +163,46 @@ export const FailedMyChatMessageContainer: ComponentSlotStyle = {
   backgroundColor: 'rgba(168, 0, 0, 0.2)'
 };
 
+/**
+ * @private
+ */
+export const chatBlockedMyMessageClasses = makeStyles({
+  body: {
+    ...defaultChatItemMessageContainer,
+    backgroundColor: 'rgb(199, 224, 244)'
+  }
+});
+
+/**
+ * @private
+ */
+export const useChatMyMessageClasses: (messageState?: MessageStatus) => Record<'body', string> = (
```

Comment:
As `useChatMyMessageClasses` and `useChatMessageClasses` do the same, should we reuse code instead?
same for `useChatFailedMyMessageClasses` and `useChatFailedMessageClasses`, `chatNormalMyMessageClasses` and `chatNormalMessageClasses`

Reply:
I moved the original logic that was in each of these out to the theme; these do indeed need to be refactored as a result.